configure

```diff
@@ -3671,6 +3680,41 @@ fi
 
 	}
 
+# Check whether --with-sim was given.
+if test "${with_sim+set}" = set; then :
+  withval=$with_sim;
+else
+  with_sim="qemu"
+
+fi
+
+{
+# Check whether --with-spike-src was given.
+if test "${with_spike_src+set}" = set; then :
+  withval=$with_spike_src;
+else
+  with_spike_src=default
+fi
+	  if test "x$with_spike_src" != xdefault; then :
+  with_spike_src=$with_spike_src
+else
+  with_spike_src="\$(srcdir)/spike"
+fi
+	}
+
+{
+# Check whether --with-pk-src was given.
+if test "${with_pk_src+set}" = set; then :
+  withval=$with_pk_src;
+else
+  with_pk_src=default
+fi
+	  if test "x$with_pk_src" != xdefault; then :
```

Comment:
Looks like a whitespace issue here.

Reply:
> Looks like a whitespace issue here.

After use autoconf, it still the same as before.
In addition, I saw the same issue at [with_qemu_src](https://github.com/riscv-collab/riscv-gnu-toolchain/blob/master/configure#L3664)
So, i think it's normal.

Reply:
Yes, I could reproduce this with autoreconf here as well.